src/Model/Anon.py

```diff
@@ -14,181 +16,114 @@
 
-    if "PatientName" in ds_rtss:
-        patient_name = str(ds_rtss.PatientName)
-        # print("Patient name - ", patient_name)
-        # MD 5 hashing
-        hash_patient_name_MD5 = uuid.uuid5(uuid.NAMESPACE_URL, patient_name)
-        # Hashing the MD5 hash again using SHA1
-        hash_patient_name_sha1 = uuid.uuid3(
-            uuid.NAMESPACE_URL, str(hash_patient_name_MD5)
-        )
-        # storing the hash to dataset
-        ds_rtss.PatientName = str(hash_patient_name_sha1)
-    else:
-        logging.warning("No patient name found in RT SS: %s", ds_rtss["SOPInstanceUID"])
-        print("NO patient Name found")
-
-    # -----------------------------------------sha1 hash for patient ID------------------------------
-
-    # if 'PatientID' in ds_rtss:
-    if "PatientID" in ds_rtss:
-        patient_ID = str(ds_rtss.PatientID)
-        # print("Patient ID - ", patient_ID)
-        # MD 5 hashing
-        hash_patient_ID_MD5 = uuid.uuid5(uuid.NAMESPACE_URL, patient_ID)
-        # Hashing the MD5 hash again using SHA1
-        hash_patient_ID_sha1 = uuid.uuid3(uuid.NAMESPACE_URL, str(hash_patient_ID_MD5))
-        # storing the hash to dataset
-        ds_rtss.PatientID = str(hash_patient_ID_sha1)
-    else:
-        print("NO patient ID not found")
-
-    # #  storing patient_name and ID in one variable
-    # if hasattribute("PatientID", ds_rtss):
-    #     P_name_ID = patient_name + " + " + patient_ID
-    # else:
-    #     P_name_ID = patient_name + " + " + "PID_empty"
-
-    # ----------------------------------------------sha1 hash for patient DOB---------------------------------------
-
-    if "PatientBirthDate" in ds_rtss:
-        patient_DOB = str(ds_rtss.PatientBirthDate)
-        # print("Patient DOB - ", patient_DOB)
-        # MD 5 hashing
-        hash_patient_DOB_MD5 = uuid.uuid5(uuid.NAMESPACE_URL, patient_DOB)
-        # Hashing the MD5 hash again using SHA1
-        hash_patient_DOB_sha1 = uuid.uuid3(
-            uuid.NAMESPACE_URL, str(hash_patient_DOB_MD5)
-        )
-        # storing the hash to dataset
-        ds_rtss.PatientBirthDate = str(hash_patient_DOB_sha1)
-    else:
-        print("Patient BirthDate not found")
-
-    # --------------------------------------------sha1 hash for patient Sex------------------------------------
-
-    if "PatientSex" in ds_rtss:
-        patient_sex = str(ds_rtss.PatientSex)
-        # print("Patient Sex - ", patient_sex)
-        # MD 5 hashing
-        hash_patient_Sex_MD5 = uuid.uuid5(uuid.NAMESPACE_URL, patient_sex)
-        # Hashing the MD5 hash again using SHA1
-        hash_patient_Sex_sha1 = uuid.uuid3(
-            uuid.NAMESPACE_URL, str(hash_patient_Sex_MD5)
-        )
-        # storing the hash to dataset
-        ds_rtss.PatientSex = str(hash_patient_Sex_sha1)
-    else:
-        print("Patient Sex not found")
+def _trim_bracketing_single_quotes(repval_string):
```

Comment:
pydicom provides a repval function / attribute that provides a string representation of an elements value.
However, it wraps the value with single quotes, which is not what the DICOM value really is.
LastName^FirstName vs. 'LastName^FirstName'
this function addresses that issue